ui/static/ui/js/learning_resources.jsx

```diff
@@ -302,9 +302,12 @@ define('learning_resources', [
               </textarea>
           </div>
           <p>
-            <button className="btn btn-lg btn-primary"
+            <button className="btn btn-lg btn-primary" id="lr_save_button"
```

Comment:
Instead of defining ids to keep track of the buttons, create two different `onClick` functions and pass an argument to `saveForm` saying whether to close the tab too. If you do this you can probably the `event` argument from `saveForm` and just call `event.preventDefault()` in both click handlers instead.